### ISSUE TITLE
Getting "curl not found" when trying to execute curl command on arc runner.

### ISSUE THREAD

**Issue by vishrane** [actions/actions-runner-controller#3130]
### What would you like added?
IS there any way to add curl package to arc runner as we need to execute curl command as part of CI.

*A clear and concise description of what you want to happen.*

Note: Feature requests to integrate vendor specific cloud tools (e.g. `awscli`, `gcloud-sdk`, `azure-cli`) will likely be rejected as the Runner image aims to be vendor agnostic.

### Why is this needed?

*A clear and concise description of any alternative solutions or features you've considered.*

### Additional context

*Add any other context or screenshots about the feature request here.*

**Comment by pavlospt** [comment]
@vishrane it is not included in the default software installed.

**Comment by vishrane** [comment]
Is there any way to install it

**Comment by pavlospt** [comment]
Yes you can use the default image as your base and create a new image for your runners!

**Comment by adefemi171** [comment]
You could do something like:

```
FROM ghcr.io/actions/actions-runner:latest

RUN sudo apt update -y && \
    sudo apt install -y \
    curl
```

**Comment by janekx21** [comment]
How do I use the created Dockerfile inside a Kubernetes cluster?
I found this thing: [example-input-for-dockerfile](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/customizing-the-containers-used-by-jobs#example-input-for-dockerfile)
and this: [updating-the-pod-specification-for-the-runner-pod](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/deploying-runner-scale-sets-with-actions-runner-controller#updating-the-pod-specification-for-the-runner-pod)

[my setup](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/quickstart-for-actions-runner-controller)

**Comment by geekflyer** [comment]
can we please add `curl` to the default image? This seems like such a common/basic thing that it's surprising this is missing.
